typescript/infra/scripts/funding/fund-keys-from-deployer.ts

```diff
@@ -409,7 +409,11 @@ class ContextFunder {
           key.context,
           key.environment,
         ).contextChainNames;
-        for (const chain of chains[role as AgentRole]) {
+        const roleToLookup = ALL_AGENT_ROLES.includes(role as AgentRole)
+          ? role
+          : Role.Relayer;
```

Comment:
would be helpful to add a comment about why `Relayer` is the fallback here

Reply:
Good point